pkg/envinfo/envinfo.go

```diff
@@ -122,20 +132,20 @@ type EnvInfoLink struct {
 
 // getEnvInfoFile first checks for the ENVINFO variable
 // then we check for directory and eventually the file (which we return as a string)
-func getEnvInfoFile(envDir string) (string, error) {
+func getEnvInfoFile(envDir string) (string, string, error) {
 	if env, ok := os.LookupEnv(envInfoEnvName); ok {
-		return env, nil
+		return env, filepath.Join(env, "..", "..", "..", "devfile.yaml"), nil
```

Comment:
lets not do this

Reply:
Indeed, let's see if I can find a better way.